cherrypy/_cpserver.py

```diff
@@ -113,6 +113,9 @@ def _set_socket_host(self, value):
     ssl_private_key = None
     """The filename of the private key to use with SSL."""
 
+    ssl_ciphers = None
+    """The ciphers list of SSL"""
+    
```

Comment:
@LenovoCindywang you added lots of trailing whitespaces to this line (not just new line), please remove them.